examples/unittests/causal_liquifier.cc

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) The JETSCAPE Collaboration, 2018
  *
  * Modular, task-based framework for simulating all aspects of heavy-ion collisions
- * 
+ *
```

Comment:
Are these changes related to your intended commits? If not they may be an artifact of migrating your branch from JETSCAPE-COMP. You may want to make a new branch from scratch, starting from JETSCAPE/release-candidate (or equivalently, JETSCAPE/master) and add your changes on top.